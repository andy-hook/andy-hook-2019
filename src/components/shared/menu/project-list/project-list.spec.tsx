import React from "react"
import ProjectList from "./project-list"
import { shallowWithTheme } from "@test-utils"
import { mockProjectsData } from "@mock-data"

describe("<ProjectList />", () => {
  it("renders correctly", () => {
    const tree = shallowWithTheme(
      "light",
      <ProjectList projects={mockProjectsData} onClick={jest.fn} open={true} />
    )
    expect(tree).toMatchSnapshot()
  })
})
