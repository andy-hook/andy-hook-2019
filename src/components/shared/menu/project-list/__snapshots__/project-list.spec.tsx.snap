// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProjectList /> renders correctly 1`] = `
<div
  className="sc-bdVaJa eYZmLV"
>
  <ul
    className="sc-bwzfXH eDFgnQ"
  >
    <li
      className="sc-htpNat eeRFYY"
    >
      <a
        className="sc-bxivhb oKECp"
        href="/projects/bright"
        onClick={[Function]}
      >
        Bright
      </a>
    </li>
    <li
      className="sc-htpNat eeRFYY"
    >
      <a
        className="sc-bxivhb oKECp"
        href="/projects/brandwatch"
        onClick={[Function]}
      >
        Brandwatch
      </a>
    </li>
    <li
      className="sc-htpNat eeRFYY"
    >
      <a
        className="sc-bxivhb oKECp"
        href="/projects/monster"
        onClick={[Function]}
      >
        Monster
      </a>
    </li>
    <li
      className="sc-htpNat eeRFYY"
    >
      <a
        className="sc-bxivhb oKECp"
        href="/projects/cymbiosis"
        onClick={[Function]}
      >
        Cymbiosis
      </a>
    </li>
    <li
      className="sc-htpNat eeRFYY"
    >
      <a
        className="sc-bxivhb oKECp"
        href="/projects/experiments"
        onClick={[Function]}
      >
        Experiments
      </a>
    </li>
  </ul>
</div>
`;
